dominion/templates/settings_js.html

```diff
@@ -1,8 +1,6 @@
 {% extends 'settings_js.html' %}{% load jsonify %}
 
 {% block settings_javascript %}
-{{ block.super }}
-WAZIMAP_COUNTRY_CODE = '{{ HURUMAP.country_code }}';
-GOOGLE_GEOCODE_API_KEY = '{{ HURUMAP.google_geocode_api_key }}';
-var MAPIT = {{ HURUMAP.mapit|jsonify|safe }};
+  {{ block.super }}
+  var dominion_countries = {{ HURUMAP.dominion_countries|jsonify|safe }};
```

Comment:
I have this in country page as `SUPPORTED_COUNTRIES`

https://github.com/CodeForAfrica/HURUmap-apps/pull/149/files#diff-109a5726e848ca027ab25046be45eca9R12

https://github.com/CodeForAfrica/HURUmap-apps/pull/149/files#diff-eb52be4d0f274502984187e2ccda30aeR98